chat_exporter/construct/message.py

```diff
@@ -243,6 +271,9 @@ async def generate_message_divider(self, channel_audit=False):
             if self.message.reference != "":
```

Comment:
` if self.message.reference != "" or self.message.interaction != "":` (check review comment as to why)